packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -244,6 +250,12 @@ describe("BlobManager", () => {
 	let runtime: MockRuntime;
 	let createBlob: (blob: ArrayBufferLike) => Promise<void>;
 	let waitForBlob: (blob: ArrayBufferLike) => Promise<void>;
+	const oldRawConfig = sessionStorageConfigProvider.value.getRawConfig;
+	let injectedSettings: Record<string, ConfigTypes> = {};
+
+	before(() => {
+		sessionStorageConfigProvider.value.getRawConfig = (name) => injectedSettings[name];
```

Comment:
I learned another way of doing this, which is when you call `loggerToMonitoringContext`, pass in a raw config provider.  I think it's slightly preferable because it's better isolation, rather than mucking with `sessionStorageConfigProvider`'s value (basically abusing the Laziness)

Reply:
Yeah, that's what is used in end-to-end tests. I did this in a separate PR - https://github.com/microsoft/FluidFramework/pull/14601

Reply:
Nice!

Reply:
But don't you need to use the same pattern here?  Or will you merge main and do it there after this PR goes in?

Reply:
Yeah, updated it.